src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  const handleClick = tab => {
+    if (tab.id !== selectedTab.id) {
+      onTabSelected(tab);
+    }
+  };
```

Comment:
let's move this into `onClick` , create separated variable for `tab.id !== selectedTab.id` and reuse in all places